### ISSUE TITLE
How to return a segmentation output tensor with gradient?

### ISSUE THREAD

**Issue by huuquan1994** [open-mmlab/mmdetection#10609]
I'm trying to utilize the outputs from a pre-trained instance segmentation model (e.g., SOLOv2) as an auxiliary loss.
Given an input image X, how can I return an output tensor Y with gradients? The current forward function will return directly the numpy masks output. 

Best,

**Comment by shreyass-ranganatha** [comment]
Any progress/updates on this? @ZwwWayne
